ord_schema/updates.py

```diff
@@ -97,22 +96,17 @@ def add_binary_identifiers(message):
         if any(identifier.type == identifier.RDKIT_BINARY
                for identifier in message.identifiers):
             continue
-        for identifier in compound.identifiers:
-            mol = None
-            if Chem and identifier.type == identifier.SMILES:
-                mol = Chem.MolFromSmiles(identifier.value)
-            elif identifier.type == identifier.INCHI:
-                mol = Chem.MolFromInchi(identifier.value)
-            elif identifier.type == identifier.MOLBLOCK:
-                mol = Chem.MolFromMolBlock(identifier.value)
-            if mol is not None:
-                source = reaction_pb2.CompoundIdentifier.IdentifierType.Name(
-                    identifier.type)
-                compound.identifiers.add(bytes_value=mol.ToBinary(),
-                                         type='RDKIT_BINARY',
-                                         details=f'Generated from {source}')
-                modified = True
-                break  # Only add one RDKIT_BINARY per Compound.
+        try:
+            mol, identifier = message_helpers.mol_from_compound(
+                compound, return_identifier=True)
+            source = reaction_pb2.CompoundIdentifier.IdentifierType.Name(
+                identifier.type)
+            compound.identifiers.add(bytes_value=mol.ToBinary(),
```

Comment:
Is it intentional that we are still excluding `add_binary_identifiers` from the default `_UPDATES`?

Reply:
Yes, we didn't want to store those in the official repo. And we don't need them for Postgres since we are already computing columns that contain Mol objects.

We might even consider removing RDKIT_BINARY as a type and dropping the bytes_value field...

Reply:
I was considering suggesting removing RDKIT_BINARY for both CompoundIdentifiers and ReactionIdentifiers. It would be nice to remove the bytes_value field from these, too. Would we still allow people to upload files on the editor but just read them as a string and insert them into the value field?

Reply:
Yes, that would make sense to me.